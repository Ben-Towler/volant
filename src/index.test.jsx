import Index from './index';

describe('App', () => {
  test('Renders without crashing', () => {
    expect(
      JSON.stringify(
        ({}, Index, { _reactInternalInstance: 'censored' }),
      ),
    ).toMatchSnapshot();
  });
});
