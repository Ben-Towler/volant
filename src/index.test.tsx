import Index from './index';

describe('App', () => {
  test('Renders without crashing', () => {
    expect(
      JSON.stringify(
        (Index),
      ),
    ).toMatchSnapshot();
  });
});
