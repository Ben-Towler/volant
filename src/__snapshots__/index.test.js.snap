// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;

exports[`renders without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
